src/js/view/controls/nextuptooltip.js

```diff
@@ -100,7 +100,7 @@ export default class NextUpTooltip {
         setTimeout(() => {
             // Set thumbnail
             this.thumbnail = this.content.querySelector('.jw-nextup-thumbnail');
-            toggleClass(this.thumbnail, 'jw-nextup-thumbnail-visible', !!nextUpItem.image);
+            toggleClass(this.content, 'jw-nextup-thumbnail-visible', !!nextUpItem.image);
```

Comment:
@aliciahurst @egreaves @hghazzi 
Please @ anyone working on DOM structure documentation about this change to the .jw-nextup-thumbnail-visible class.